spec/ruby/optional/capi/ext/kernel_spec.c

```diff
@@ -153,6 +153,10 @@ VALUE kernel_spec_rb_rescue(VALUE self, VALUE main_proc, VALUE arg,
   rb_ary_push(main_array, main_proc);
   rb_ary_push(main_array, arg);
 
+  if (raise_proc == Qnil && arg2 == Qnil) {
```

Comment:
There's hopefully be a better way of doing this, but `rb_funcall` gets upset when I pass it null values so this is my way of bypassing.

Reply:
That's fine, thanks for adding the spec.